apps/test/unit/lib/levelbuilder/lesson-editor/LevelTokenDetailsTest.js

```diff
@@ -57,6 +65,21 @@ describe('LevelTokenDetails', () => {
     assertChecked(wrapper, 'challenge', false);
   });
 
+  it('bonus is disabled if lesson extras are not available for script', () => {
+    const wrapper = shallow(<LevelTokenDetails {...defaultProps} />);
+    assertDisabled(wrapper, 'bonus', true);
+  });
+
+  it('bonus is enabled if lesson extras are available for script', () => {
+    const wrapper = shallow(
+      <LevelTokenDetails
+        {...defaultProps}
+        lessonExtrasAvailableForScript={true}
+      />
+    );
+    assertDisabled(wrapper, 'bonus', false);
+  });
```

Comment:
If the checkbox is checked and disabled, the curriculum writer would have to enable lesson extras on the script in order to be able to disable this checkbox. one lightweight solution would be to only disable the checkbox if this.props.scriptLevel.bonus is also false. another which seems harder could be to make it so that disabling the checkbox on the script page clears this setting for all its script levels.

I'm not sure how likely we are to run into this, but since the first option seems easy I'd recommend going with that.